### PR TITLE
Mx93 som

### PR DESCRIPTION
**Pull request by nsdrude**

### CODE REVIEW

**Comment by nsdrude** [comment]
@eranmati please review the changes? It's working on my SoM, see starting at **Part number: VSM-MX93-101**:

```
U-Boot SPL 2022.04-lf_v2022.04_var02+g277ff23bfc (Feb 21 2023 - 21:46:06 +0000)
SOC: 0xa0009300
LC: 0x40010
M33 prepare ok
Normal Boot
Trying to boot from BOOTROM
Boot Stage: Primary boot
image offset 0x8000, pagesize 0x200, ivt offset 0x0
Load image from 0x48000 by ROM_API
NOTICE:  BL31: v2.6(release):lf-5.15.71-2.2.0-0-g3c1583ba0
NOTICE:  BL31: Built : 11:00:38, Nov 21 2022


U-Boot 2022.04-lf_v2022.04_var02+g277ff23bfc (Feb 21 2023 - 21:46:06 +0000)

CPU:   i.MX93(52) rev1.0 1700 MHz (running at 1692 MHz)
CPU:   Consumer temperature grade (0C to 95C) at 50C
Reset cause: POR (0x1)
Model: Variscite VAR-SOM-MX93 on Symphony-Board
DRAM:  2 GiB
Core:  192 devices, 23 uclasses, devicetree: separate
MMC:   FSL_SDHC: 0, FSL_SDHC: 1
Loading Environment from MMC... *** Warning - bad CRC, using default environment

In:    serial
Out:   serial
Err:   serial

BuildInfo:
  - ELE firmware version 0.0.8-80d3db4b

PTN5150: Vendor ID [0x3], Version ID [0x1], Addr [I2C4 0x3d]

Part number: VSM-MX93-101
Assembly: AS0123456789
Production date: 2023 Jan 22
Serial Number: f8:dc:7a:7a:1d:db
SOM revision: 0x1
DRAM PN: VIC-1032
switch to partitions #0, OK
mmc1 is current device
UID: 0xfe6f9252 0x4c461fc3 0x741b488c 0x2cb95986
flash target is MMC:1
Net:   Get shared mii bus on ethernet@42890000
ADIN1300 PHY detected at addr 5
eth1: ethernet@42890000, eth0: ethernet@428a0000 [PRIME]
Fastboot: Normal
Normal Boot
Hit any key to stop autoboot:  0 
```

Also, you said I should add support to: https://github.com/varjig/imx_kit_test/blob/master/mx8mm_vareeprom_dram.sh

But it looks like it currently only supports the mini. The EEPROM file can be written by following the readme here: https://github.com/varjig/eeprom-tools/tree/imx93-var-som/imx93_eeprom

**Comment by nsdrude** [comment]
@eranmati the latest can be viewed here: https://github.com/varjig/imx_kit_test/tree/mx93-som

File: mx8m_vareeprom.sh [hunk 1]
```diff
@@ -1,51 +1,102 @@
...
+	# DRAM size offset and size
+	DS_OFFSET=0x26
+	DS_SIZE=1
+
+	# DDR CRC32
+	DDR_CRC_OFFSET=0x2c
+	DDR_CRC_SIZE=0x4
+
+	# DDR VIC Part Number
+	DDR_VIC_OFFSET=0x30
```
Comment (eranmati) [inline mx8m_vareeprom.sh:49]: Sometimes it is referred to as DRAM and sometimes as DDR - I think this is confusing (think of searching DDR_VIC or DRAM_VIC for example) and should be aligned to one of them.

Comment (nsdrude) [reply mx8m_vareeprom.sh]: Fixed by https://github.com/varjig/imx_kit_test/pull/1/commits/273d2625b9dc96a56217f54e83dc77655c5ecb01

**Pull request merged**